pkg/lib/bundle/generate.go

```diff
@@ -298,6 +302,10 @@ func ValidateChannelDefault(channels, channelDefault string) (string, error) {
 	var chanErr error
 	channelList := strings.Split(channels, ",")
 
+	if len(channelList) == 0 || slice.ContainsString(channelList, "", nil) {
+		return chanDefault, fmt.Errorf("invalid channels is provied: %s", channels)
```

Comment:
This error message seems confusing. Could we be more descriptive of what is actually going wrong here? It seems like we are actually checking two things that should have separate error messages:

1. That no channels were specified
2. That the list of channels included an empty string in the list I think?

Also spelling/grammar nit: use `are` instead of `is` for plurals, and `provided` instead of `provied`

Reply:
ha, I copied that misspell and didn't even see it https://github.com/operator-framework/operator-registry/blob/a25687c90bbacce3c200a232376fa4952d22d307/pkg/lib/bundle/generate.go#L319

Reply:
https://play.golang.org/p/uemuPhlTisX found out that this split will always output an array with at least 1 length. My bad.